README.md

```diff
@@ -821,64 +830,67 @@ class C {
 }
 ```
 
-Would produce the following two metadata objects:
+You would be able to access the metadata for this class like so:
 
 ```js
-let staticMeta = {
-  public: {
-    constructor: { "my-meta": true, [MY_META]: true },
-    x: { "my-meta": true, [MY_META]: true },
-    y: { "my-meta": true, [MY_META]: true },
-  },
+C[Symbol.metadata][MY_META];
+// {
+//   constructor: 'metadata',
+
+//   public: {
+//     x: 'metadata',
+//     y: 'metadata',
+//   },
+
+//   private: ['metadata']
+// }
+
+C[Symbol.metadata][MY_META];
+// {
+//   public: {
+//     a: 'metadata',
+//     b: 'metadata',
+//   },
+
+//   private: ['metadata']
+// }
+```
 
-  private: {
-    "#z": { "my-meta": true, [MY_META]: true },
-  }
-}
+Subsequent definitions to the same key will result in an array of values instead of a single value. This allows metadata to be built up with multiple decorators without having to _read_ the metadata during decoration.
 
-let nonStaticMeta = {
-  public: {
-    a: { "my-meta": true, [MY_META]: true },
-    b: { "my-meta": true, [MY_META]: true },
-  },
+```js
+const VALIDATIONS = Symbol();
 
-  private: {
-    "#c": { "my-meta": true, [MY_META]: true },
-  }
+function validateString(value, context) {
+  context.defineMetadata(VALIDATIONS, (value) => typeof value === "string"));
```

Comment:
This is a good point, we should make it so that calling the function any time after the metadata has been reified and assigned to the class is an error.